quests/the_forge.md

```diff
@@ -250,48 +258,52 @@ class pasture (
 }
 ```
 
+<div class = "lvm-task-number"><p>Task 5:</p></div>
+
 Next, edit the `pasture_config.yaml.epp` template. We'll use a conditional
-statement to only include the `:db_uri:` setting if there is a value set for
-the `$db_uri` variable. This will allow us to leave this option unset and use
+statement to only include the `:db:` setting if there is a value set for
+the `$db` variable. This will allow us to leave this option unset and use
 the Pasture application's own default if we haven't explicitly set the
-`pasture` class's `db_uri` parameter.
+`pasture` class's `db` parameter.
 
 ```puppet
 <%- | $port,
       $default_character,
       $default_message,
       $sinatra_server,
-      $db_uri,
+      $db,
 | -%>
 # This file is managed by Puppet. Please do not make manual changes.
 ---
-  :default_character: <%= $default_character %>
-  :default_message: <%= $default_message %>
-  <% if $db { -%>
-  :db: <%= $db %>
-  <% } -%>
-  :sinatra_settings:
-    :port:   <%= $port %>
-    :server: <%= $sinatra_server %>
+:default_character: <%= $default_character %>
+:default_message: <%= $default_message %>
+<%- if $db { -%>
```

Comment:
These are the only lines that need to not be indented, but I don't think you want the <%-. I'm not sure.  I fixed it by just unindenting these lines